### ISSUE TITLE
Unexpected trailing comma for spread operator

### ISSUE THREAD

**Issue by DeyLak** [eslint/eslint#8289]
**Tell us about your environment**

* **ESLint Version: 3.18.0**
* **Node Version: 6.2.1**
* **npm Version: 3.9.3**

**What parser (default, Babel-ESLint, etc.) are you using?**
"babel-eslint": "^7.1.1"
**Please show your full configuration:**

```
module.exports = {
  parser: 'babel-eslint',
  env: {
    browser: true,
  },
  rules: {
    'comma-dangle': [
      2,
      'always-multiline',
    ],
  },
}
```

**What did you do? Please include the actual source code causing the issue.**

```js
const func = ({
  first = 1,
  second = 2,
  ...other,
}) => {
}
```

**What did you expect to happen?**
No errors
**What actually happened? Please include the actual, raw output from ESLint.**
 33:11  error  Unexpected trailing comma  comma-dangle

Thank you

**Comment by not-an-aardvark** [comment]
Thanks for the report. However, this working as intended. The [language spec](https://github.com/sebmarkbage/ecmascript-rest-spread) disallows trailing commas after rest properties.

Also see #8273, #7857, #7614, #7502, #7463, #7422, #7297, and #7298.

**Comment by ilyavolodin** [comment]
Closing, as this is behaving correctly.
